keyboards/heliar/wm1_hotswap/wm1.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2019 HELIAR MK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+#define xxx KC_NO
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+
+#define LAYOUT( \
+	K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, \
+	K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E, \
+	K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2C, K2D, \
+	K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D,      \
+	K40,      K41, K42, K43,      K44,           K45,      K46, K47, K48, K49  \
+) \
+{ \
+	{K00, K01,   K02,   K03, K04, K05, 	K06,	K07,	K08,	K09, K0A,   K0B, K0C,   K0D,	K0E}, \
+	{K10, K11,   K12,   K13, K14, K15, 	K16,	K17,	K18,	K19, K1A,   K1B, K1C,   K1D,	K1E}, \
+	{K20, K21,   K22,   K23, K24, K25, 	K26,	K27,	K28,	K29, K2A,   K2B, KC_NO,	K2C,    K2D}, \
+	{K30, K31,   K32,   K33, K34, K35, 	K36,	K37,	K38,	K39, K3A,   K3B, K3C,   K3D,	KC_NO}, \
+	{K40, KC_NO, K41, 	K42, K43, KC_NO,K44, 	KC_NO,	KC_NO,	K45, KC_NO, K46, K47,   K48,	K49}  \
+}
```

Comment:
Could you please change Lines 31-44 to:

```c
#define LAYOUT( \
    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D, K0E, \
    K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E, \
    K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2D, K2E, \
    K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D,      \
    K40,      K42, K43, K44,      K46,           K49,      K4B, K4C, K4D, K4E  \
) \
{ \
	{   K00, K01,   K02, K03, K04, K05,   K06, K07,   K08,   K09, K0A,   K0B, K0C,   K0D, K0E   }, \
	{   K10, K11,   K12, K13, K14, K15,   K16, K17,   K18,   K19, K1A,   K1B, K1C,   K1D, K1E   }, \
	{   K20, K21,   K22, K23, K24, K25,   K26, K27,   K28,   K29, K2A,   K2B, KC_NO, K2D, K2E   }, \
	{   K30, K31,   K32, K33, K34, K35,   K36, K37,   K38,   K39, K3A,   K3B, K3C,   K3D, KC_NO }, \
	{   K40, KC_NO, K42, K43, K44, KC_NO, K46, KC_NO, KC_NO, K49, KC_NO, K4B, K4C,   K4D, K4E   }  \
}
```

Ideally, the keys should be numbered by *electrical* position rather than physical, which means every position that shares the same middle character (`2` for instance) is on the same row in the switch matrix, and every position that ends in the same character is on the same matrix column.

This code block is the same as yours when compiled, but makes it easier to debug if problems are found later.